src/alchemlyb/parsing/amber.py

```diff
@@ -254,61 +246,91 @@ def extract_u_nk(outfile, T):
 
     Returns
     -------
-    u_nk : DataFrame
-        Reduced potential for each alchemical state (k) for each frame (n).
-
-
-    .. versionchanged:: 0.5.0
-        The :mod:`scipy.constants` is used for parsers instead of
-        the constants used by the corresponding MD engine.
-
+    Dict
+        A dictionary with keys of 'u_nk', which is a pandas DataFrame of reduced potentials for each
+        alchemical state (k) for each frame (n), and 'dHdl', which is a Series of dH/dl
+        as a function of time for this lambda window.
```

Comment:
.. versionadded:: 1.0.0

Reply:
Great, done!

Reply:
Thanks. Have you checked that the doc looks good? I was under the impression that the .. versionadded:: 1.0.0 need to blank line before it but I might be wrong.